content/blog/2022-05-18-learned-federated-query-optimization.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "A Learned Query Compiler"
+[extra]
+latex = true
+[[extra.authors]]
+name = "Victor Giannakouris"
+link = "https://www.linkedin.com/in/vgian/"
++++
+
+# What was the goal?
+The goal of this project was to create an optimizer for a cross-database query compiler. A cross-database query, also 
+known as federated query, is an SQL query that consists of tables that span across different, individual database 
```

Comment:
Consider italicizing new terms the first time you introduce them, like *federated query*.

Reply:
done